doc/ref/overview.xml

```diff
@@ -128,6 +130,7 @@ recent versions ready for release at this time, new packages and new
 versions may be released at any time and can be easily installed in
 your copy of &GAP;.
 <P/>
+<!-- TODO update this paragraph -->
 With &GAP; there are two packages (the library of ordinary and
```

Comment:
I guess this should be changed to "several packages" and also include the smallgrp, primgrp, transgrp.

Reply:
I agree.